packages/core/src/utils/ValidJWEStucture.ts

```diff
@@ -0,0 +1,5 @@
+import type { EncryptedMessage } from '../types'
+
+export function validJWEStucture(message: any): message is EncryptedMessage {
```

Comment:
For methods that return booleans we usually start with isXXX. Also for abbreviations (JWE) we only capitalize the first letter (so Jwe)

```suggestion
export function isValidJweStucture(message: any): message is EncryptedMessage {
```